portfolio/src/main/java/com/google/sps/servlets/ListImageServlet.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.servlets;
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Shows all of the images uploaded to Cloud Storage. */
+@WebServlet("/list-images")
+public class ListImageServlet extends HttpServlet{
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      // List all of the uploaded files.
+      String projectId = "ysun-sps-summer22";
+      String bucketName = "ysun-sps-summer22.appspot.com";
+      Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
+      Bucket bucket = storage.get(bucketName);
+      Page<Blob> blobs = bucket.list();
```

Comment:
I would move the above block to a separate method. Something like
Page<BLob> getImages();